admin_manual/release_notes.rst

```diff
@@ -11,6 +12,45 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.3_release_notes_label:
+
+Changes in 10.0.3
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues of ownCloud Server 10.0.3 that need your attention:
+
+**The full ownCloud Server 10.0.3 changelog can be found here: https://github.com/owncloud/core/blob/stable10/CHANGELOG.md**
+
+* It is now possible to directly upgrade from 8.2.11 to 10.0.3 in a single upgrade process.
+* Added occ command to list routes which can help administrators setting up network firewall rules.
+* 'occ upgrade' is now verbose by default.
+    * Administrators may need to adjust scripts for automated setup/upgrade procedures that rely on 'occ upgrade' outputs.
```

Comment:
since this is the only sub-item, I suggest putting it on the same row

Reply:
changed